app/view/WarningView.js

```diff
@@ -424,7 +424,7 @@ SDL.warningView = Em.ContainerView
             var self = this;
             this._parentView.set(
               'fade', this
-                .checkForCCS3AnimationSupport()

```

Comment:
@wolfylambova22 , there is CSS, not "Customer Connectivity Settings ". 
There is mistake, not CCS, must be CSS (Cascading Style Sheets)
Please change to CSS.